__tests__/app.test.js

```diff
@@ -104,4 +104,41 @@ describe("/api/articles", () => {
                 expect(articles).toBeSortedBy("created_at", { descending: true })
             })
     })
+})
+
+describe("/api/articles/:article_id/comments", () => {
+    test("200: GET responds with an array of comments for a given article id, with most recent comment first", () => {
+        return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({body}) => {
+            const commentsArray = body.comments
+            expect(commentsArray.length).toBe(11)
+            expect(commentsArray).toBeSortedBy("created_at",  { descending: true })
+            commentsArray.forEach(comment => {
+                expect(typeof comment.comment_id).toBe("number")
+                expect(typeof comment.votes).toBe("number")
+                expect(typeof comment.created_at).toBe("string")
+                expect(typeof comment.author).toBe("string")
+                expect(typeof comment.body).toBe("string")
+                expect(typeof comment.article_id).toBe("number")
+            })
```

Comment:
This is fine and works.
Going forward you might want to look into expect objectContaining or toMatchObject jest matchers. They allow for much more robust testing.